web/src/stream_popover.js

```diff
@@ -414,6 +414,12 @@ export async function build_move_topic_to_stream_popover(
         send_notification_to_new_thread = send_notification_to_new_thread === "on";
         send_notification_to_old_thread = send_notification_to_old_thread === "on";
         current_stream_id = Number.parseInt(current_stream_id, 10);
+        select_stream_id = Number.parseInt(select_stream_id, 10);
```

Comment:
`select_stream_id` is of type string.
Even though this line is added now, it should have already been present, because a few lines later, we do a strict equality check on `select_stream_id` and `current_stream_id`, which would never be true in absence of this line.

Reply:
Since this sounds like a bug fix for an existing bug, do you want to put it in a separate prep commit with its own commit message in this PR? That's generally how we manage this sort of fix noticed while working on something.

Reply:
I have separated it into its own commit.